bundles/CoreBundle/src/DependencyInjection/Configuration.php

```diff
@@ -604,6 +604,10 @@ private function addAssetNode(ArrayNodeDefinition $rootNode): void
                                 ->defaultTrue()
                                 ->info('Scan PDF documents for unsafe JavaScript.')
                             ->end()
+                            ->enumNode('open_pdf_in_new_tab')
+                                ->values(['all-pdfs', 'only-unsafe', 'none'])
+                                ->defaultValue('all-pdfs')
```

Comment:
```suggestion
                                ->defaultValue('only-unsafe')
```
I'd prefer to have a security fist default value here.